backend/backend.go

```diff
@@ -1018,6 +1018,7 @@ func (backend *Backend) SystemOpen(url string) error {
 
 	whitelistedPatterns := []string{
 		"^https://shiftcrypto.ch/",
+		"^https://s.shiftcrypto.ch/",
```

Comment:
so, do we use `s.shiftcrypto` or `x.shiftcrypto` in the end?

Reply:
yes please if you don't mind.

Reply:
fixed in 3d062feb84ae66e2891ae155941f6d3fd00f2d18